src/App.css

```diff
@@ -22,6 +22,11 @@
   font-size: large;
 }
 
+.customer_details:hover {
+  background-color: aqua;
```

Comment:
changes colour to a darker colour on hover for easier readability, focus and nice UI effect